content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -109,6 +109,10 @@ to manage the lifecycle of Istio.
     $ helm install install/kubernetes/helm/istio --name istio --namespace istio-system --set global.mtls.enabled=true
     {{< /text >}}
 
+## Installation Options
+
+To customize Istio and install add-ons, use the --set <key>=<value> option in the helm template or install command. See [Installation Options](/docs/reference/config/installation-options/) for the list of supported keys.
+    
```

Comment:
I believe this text would better format as sub-bullets to the Install Istio in your cluster parent bullet.  This would also reduce the complexity of this sentence for ESL individuals.

Reply:
please place `--set <key>=<value>` in backticks.

Reply:
Replace the word See with Reference or restructure the sentence such that it is simpler.  E.g. `[Installation Options](/docs/reference/config/installation-options/) references supported installation key and value pairs.`

Reply:
maybe put this blob before asking user to choose either of the mutually exclusive options?   

would be good to mention they can always add an install option using `helm upgrade` as that is a more common pattern.

Reply:
true, it has to rerun the helm template.